framework/include/systems/NonlinearSystemBase.h

```diff
@@ -145,6 +147,26 @@ class NonlinearSystemBase : public SystemBase, public PerfGraphInterface
                          const std::string & name,
                          InputParameters & parameters);
 
+  /**
+   * Adds a hybridized kernel
```

Comment:
```suggestion
   * Adds a Hybridized Discontinuous Galerkin kernel
```

Reply:
feel free to disregard this and the next few comments if hybridized is already implies DG